### ISSUE TITLE
Bot initialization: Add initialization and validation flow for startup

### ISSUE THREAD

**Issue by Evalir** [aracred/bot#7]
The bot needs certain environment variables so it can properly start up. If this doesn't affect heroku's behavior on deployment, we should introduce checks for the required environment variables and either warn the user about the lack of them, or error and exit if we detect something's missing.

**Comment by burrrata** [comment]
Is this resolved now that we have the [Heroku button we can click on the README](https://github.com/aracred/bot/)?

**Comment by Evalir** [comment]
Not quite; while it is mostly solved, having this would be nice for debugging purposes. Let's leave this as an open issue to see if we can come back to it later, or a community contributor wants to take it up.
